pkg/daemon/daemon.go

```diff
@@ -413,7 +413,9 @@ func (dn *Daemon) handleErr(err error, key interface{}) {
 		glog.Fatalf("Error handling node sync: %v", err)
 	}
 
-	dn.updateErrorState(err)
+	if err := dn.updateErrorState(err); err != nil {
+		glog.Infof("Could not update annotation: %v", err)
```

Comment:
Since we're logging an error, we should treat it like an error. Use `glog.Error()` instead of `glog.Infof()`.